tests/maptovariantconverter/maptovariantconverter.pro

```diff
@@ -0,0 +1,22 @@
+include(../../src/libtiled/libtiled.pri)
+
+QT += testlib
+CONFIG += c++14
+TEMPLATE = app
+
+macx {
+    LIBS += -L$$OUT_PWD/../../bin/Tiled.app/Contents/Frameworks
+} else {
+    LIBS += -L$$OUT_PWD/../../lib
+}
+
+!win32:!macx:!cygwin {
+    QMAKE_RPATHDIR += \$\$ORIGIN/../../lib
+
+    # It is not possible to use ORIGIN in QMAKE_RPATHDIR, so a bit manually
+    QMAKE_LFLAGS += -Wl,-z,origin \'-Wl,-rpath,$$join(QMAKE_RPATHDIR, ":")\'
+    QMAKE_RPATHDIR =
+}
+
+# Input
+SOURCES += test_maptovariantconverter.cpp
```

Comment:
It's great that you're adding test cases for this new functionality! I've been very bad at making sure any functionality is property tested...

However, I would say we shouldn't bother adding qmake projects for the tests. In fact I've only been keeping the qmake projects around since the Tiled snap is still built with qmake, but it is otherwise no longer used (except for packaging for various distributions). I've just pushed a change that makes the tests easily runnable using Qbs (334cb15b7b19c5db5fb9f0b710197a5d9844daeb).

Reply:
Alright, I'll remove .pro files.
Anyway, I was thinking there should be one test project for libtiled with one test suite running tests of lots of classes. That way test_mapreader test_mapwriter etc... would be in one test project and run at once.